apps/sso/components/reset-card/reset-card.tsx

```diff
@@ -100,6 +106,26 @@ const StyledForm = styled.form`
   padding: 10px 20px;
 `;
 
+const Input = styled.input`
+  border: solid 0.1rem #bcbcbc;
+  background-color: #f8f8f8;
+  border-radius: 0.3rem;
+  padding: 10px;
+  padding-left: 15px;
+  font-family: InterVariable, sans-serif;
+  font-size: 1.1rem;
+  color: #676767;
+  width: 75%;
+  margin-top: 1rem;
+  ::placeholder {
+    color: #bcbcbc;
+  }
+  ::-ms-input-placeholder {
+    /* Microsoft Edge */
+    color: #bcbcbc;
+  }
+`;
+
```

Comment:
Should add special outline when user clicks on the input (:active selector). On prod the implementation makes the border purple I think

Reply:
Not done, closing anyways